changelogs/fragments/187-role-errors.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "When parsing errors happen during in the Sphinx extension, the extension now emits error messages during the build process next to emitting error markup (https://github.com/ansible-community/antsibull-docs/pull/187)."
```

Comment:
The actual code looks good to me but the changelog fragment doesn't parse:

> happen during in the Sphinx extensions

Reply:
```suggestion
  - "When parsing errors happen in the Sphinx extension, the extension now emits error messages during the build process next to emitting error markup (https://github.com/ansible-community/antsibull-docs/pull/187)."
```

Reply:
I hope this way it's better.